scripts/build_android.sh

```diff
@@ -103,6 +103,14 @@ fi
 # Don't build artifacts we don't need
 CMAKE_ARGS+=("-DBUILD_TEST=OFF")
 CMAKE_ARGS+=("-DBUILD_BINARY=OFF")
+
+# If there exists env variable and it equals to 1, build lite interpreter.
+# cmd:  BUILD_LITE_INTERPRETER=1 ./scripts/build_android.sh
+if [ "${BUILD_LITE_INTERPRETER}" == 1 ]; then
```

Comment:
Do we have the same changes in build_ios.sh?

Reply:
Good point! I add the change in `build_ios.sh` in the latest commit and update the test plan. I'm not sure how to test on ios device though. Any suggestion? Also do you know if it's possible to use different CMake flag in CI test? I'm thinking to add the test with similar behavior like this:
```
BUILD_PYTORCH_MOBILE=1 IOS_PLATFORM=SIMULATOR BUILD_LITE_INTERPRETER=1   ./scripts/build_ios.sh
BUILD_PYTORCH_MOBILE=1 IOS_PLATFORM=SIMULATOR BUILD_LITE_INTERPRETER=0   ./scripts/build_ios.sh
```